File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/accounting/transactions/shared/DTOs/subledger-dto.ts
export class SubLedger_DTO {
    public SubLedgerId: number = 0;
    public SubLedgerName: string = "";
    public SubLedgerCode: string = "";
    public LedgerId: number = 0;
    public LedgerName: string = "";
    public Description: string = "";
    public IsActive: boolean = true;
    public OpeningBalance: number = 0;
    public DrCr: boolean = true;
    public HospitalId: number = 0;
    public IsDefault: boolean = false;
}

File: Code/Websites/DanpheEMR/Services/Accounting/DTOs/SubLedger_DTO.cs
using System;

namespace DanpheEMR.Services.Accounting.DTOs
{
    public class SubLedger_DTO
    {
        public int SubLedgerId { get; set; }
        public string SubLedgerName { get; set; }
        public string SubLedgerCode { get; set; }
        public int LedgerId { get; set; }
        public string LedgerName { get; set; }
        public string Description { get; set; }
        public bool IsActive { get; set; }
        public decimal OpeningBalance { get; set; }
        public bool DrCr { get; set; }
        public int HospitalId { get; set; }
        public bool IsDefault { get; set; }
    }
}
